privacy.php

```diff
@@ -24,21 +24,13 @@
                 <h4>Sharing Data</h4>
                 <p>elementary OS does not share any data collected from your online accounts. The data is never sent to elementary or third-party servers.</p>
             </div>
-            <div class="row docs">
-                <h2>Blog</h2>
-                <p>Our <a href="https://blog.elementary.io" target="_blank">blog</a> does not use any external scripts, cookies, or other tracking methods.</p>
-            </div>
             <div class="row docs">
                 <h2>elementary.io</h2>
-                <h3>Do Not Track</h3>
-                <p>This site honors the do-not-track setting found in most modern browsers and disables Google Analytics when it is present. We cannot automatically remove other tracking methods without severely impacting the use of the site. Should you wish to manually manage these tracking methods, reference the section on disabling cookies.</p>
                 <h3>Cookies</h3>
                 <p><strong>You can choose to disable or selectively turn off any cookies or third-party cookies in your browser settings.</strong></p>
                 <p>This site uses cookies for incremental improvements. You may find the services function without them but at a reduced usability. For example, the site will not remember if you have previously paid for elementary OS; by default you will be asked to pay again.</p>
                 <h5 data-l10n-off>Cloudflare</h5>
                 <p>Stores cookies to log behavioral elements and analyze potential threats. For more information, see the <a class="read-more" target="_blank" rel="noopener" href="https://www.cloudflare.com/security-policy">Cloudflare Privacy &amp; Security Policy</a></p>
```

Comment:
For transparency, we should add something like this:

```suggestion
                <p>Stores cookies to log behavioral elements and analyze potential threats. For more information, see the <a class="read-more" target="_blank" rel="noopener" href="https://www.cloudflare.com/security-policy">Cloudflare Privacy &amp; Security Policy</a></p>
                <h5 data-l10n-off>Plausible</h5>
                <p>We use the open source Plausible Analytics routed through our <code>stats.elementary.io</code> domain to count website visits, downloads, etc. No personal data—not even an IP address or browser user agent—is stored. <a class="read-more" target="_blank" rel="noopener" href="https://plausible.io/data-policy">Plausible Data Policy</a></p>
```

Reply:
Yeah I like this. Should we try to fit in a link to the dashboard here too or naw?

Reply:
Oh yeah good call. Updated.